data/settings/icon-grid-C.json.in

```diff
@@ -36,29 +39,8 @@
     "net.sourceforge.Extremetuxracer.desktop",
     "net.sourceforge.Warmux.desktop"
   ],
-  "eos-folder-work.directory" : [
-    "libreoffice-calc.desktop",
-    "libreoffice-impress.desktop",
-    "gnome-calculator.desktop",
```

Comment:
I'm adding translation back - but yes we're dropping resume and finance - their usage is very low. There's not currently a sane place to put the calculator and I don't want it on top level as again, usage is low.